src/common/middleware/usercheck.js

```diff
@@ -1,16 +1,24 @@
 const CookieModel = require('../../common/models/Cookie.class');
 const logger = require('../utils/logger')(__filename);
 
+const checkUserInCookie = (cookie) => {
```

Comment:
User check middleware now factors in the verified status of the user and the role of the user in the cookie. If they are not there, then the app considers the user not logged in.